Dockerfile

```diff
@@ -1,14 +1,12 @@
 ARG FUNCTION_DIR="/home/app/"
-ARG RUNTIME_VERSION="3.9"
-ARG DISTRO_VERSION="3.16"
+ARG RUNTIME_VERSION="3.12"
+ARG DISTRO_VERSION="3.22"
 
 FROM python:${RUNTIME_VERSION}-alpine${DISTRO_VERSION} AS python-alpine
 RUN apk add --no-cache \
     bash               \
-    curl               \
```

Comment:
This is the shared image and these dependencies end up in the runtime image (which we don't need) so I've moved them into the builder